.travis.yml

```diff
@@ -34,3 +35,17 @@ jobs:
     - env: PACKAGE=SDPA
     - env: PACKAGE=SDPAFamily
     - env: PACKAGE=Tulip
+    - env:
+      - PACKAGE=Xpress
+      - secure: rVoIRTheBcPJ7Sytnvu8pje0Qds4ekWmdb7cEI0rRE+8s4+4M4aNjExjTQsDgFph10L/O5FCVwDP0EgmUWzU2zmZ4HsR7rLEDp5obzhHWX6qS04xFXiZUrwNjLaP01k+BUMDNsAJprXJ00Q7jCucPCWokKp4fPFZWRxOp8cmStWeVbDemnbHJGJlqqnFre1sL9g6MOQHue9fGN39d/LDU/9bkXHjuk4WooeMTohXzJT7KoKtC9PAv9+5j5Sup5nRySlJaTj/pc0iAHRwHafE6pK4d6jIgl/RPy9oqjahqhVtfO7DcUxmpcuYliQgpJknWZkKhIG4+o7jaWftQcpBAIJPK2XYsUXVDs2TDwoJc4jlkoq3WZ+LOqagx7zaCYM9cSEZ1Uj8EXE0sp/6brEshoOH8tIjvHNegii2tEkWrsQXHD/Lqhvso6gNOCu7u3826s2QjuygsgUR70tJGKYBqZTp4D07UIQu8/SDLSRl7MlpcE2r2Wqkiaic1+h/EUOiy0CD9LVGbC40qtsp0PHDOp8R60PwqIWO882TRA+65roCB3o/Lic9OzvAFuiyPihEsqA4xBOGp6xPKA9ALSRaxNq0OfUXLcdihNApSDNySDo9g1e7Y78WH6p9co3q0/+d8E6dC/Tvpci2fepfKC6izVJhx9oAdV1P04pJdVrZcsw=
+      os:  osx
+before_install:
+  - if [[ $TRAVIS_OS_NAME == "osx" ]]; then
```

Comment:
Can you add a comment explaining that this is for Xpress ?

Reply:
moved inside xpress